Mathlib/Analysis/NormedSpace/Exponential.lean

```diff
@@ -84,7 +84,7 @@ variable {𝔸}
 It is defined as the sum of the `FormalMultilinearSeries` `expSeries 𝕂 𝔸`.
 
 Note that when `𝔸 = Matrix n n 𝕂`, this is the **Matrix Exponential**; see
-[`Analysis.NormedSpace.MatrixExponential`](../MatrixExponential) for lemmas specific to that
+[`Analysis.NormedSpace.MatrixExponential`](Mathlib/Analysis.NormedSpace.MatrixExponential) for lemmas specific to that
```

Comment:
This mix of dots and slashes doesn't look like it would work to me. I'm guessing it should just have slashes. What does wrong with the old link?

Reply:
I think the old link had an extra `.`. The following should work
```suggestion
[`Analysis.NormedSpace.MatrixExponential`](./MatrixExponential) for lemmas specific to that
```

Reply:
I think the old link had an extra `.`. The following should work
```suggestion
[`Analysis.NormedSpace.MatrixExponential`](./MatrixExponential) for lemmas specific to that
```